### ISSUE TITLE
Periodically the target animation will stay on a token and any targets after that will use the default graphics

### ISSUE THREAD

**Issue by Zamrod** [eadorin/target-enhancements#17]
Sometimes when I leave a token targeted for two long (or maybe after I alt-tab out and alt-tab back in?) the graphics from the Target Enhancements will stay on a token when when it isn't targeted. From that point onward targeting anything will use the default graphics and won't trigger any of the other features of TE (no icons appear).

**Comment by Kayne** [comment]
Yup, that's happening everytime with enabled Combat Utility Belt and Minor Quality of Life Improvements.

When you select a target, execute a roll (like attack or spell), the "Target Enchanemented" indicator stays on token forever. When you try to select different token, or even the same, it is targeted but no visual changes are applied and no smaller tokens are displayed on targeted tokens anymore.

**Comment by HumanWreck** [comment]
Setting "Untarget at end of a turn" in Minor Quality of Life Improvements to "never" fixed this issue for me. I have also CUB installed.
